.goreleaser.yml

```diff
@@ -19,40 +14,38 @@ builds:
 - &pulumibin
   id: pulumi
   binary: pulumi
+  dir: pkg
+  main: ./cmd/pulumi
+  gobinary: ../scripts/go-wrapper.sh
+  env:
+  - CGO_ENABLED=0
+  - GO111MODULE=on
   goos: ['linux', 'darwin', 'windows']
   goarch: ['amd64', 'arm64']
   goamd64: ['v1']
-  ignore:
-    - goos: windows
```

Comment:
Do we build windows/arm64 now?

Reply:
We can & should - it's a platform Go supports and Windows, Windows Server support them as platforms.